docs/training-guide.md

```diff
@@ -120,7 +120,7 @@ because they are relatively clean, and the pipeline supports automatic downloadi
     ...
 ```
 
-### Custom datasets
+### Custom local datasets
```

Comment:
Not for Taskcluster though right? Datasets on local disk can only be used from Snakemake local / slurm + NFS setup.  We should replace this with docs for the bucket importer.

Reply:
I did this before our meeting on Snakemake status. I'll update this to only speak to taskcluster.